elastic/search_query.go

```diff
@@ -76,7 +77,11 @@ func (ec *EventsCursor) searchQuery() ([]byte, error) {
 	if ec.newestIngested.IsZero() {
 		docrange.Range[fn.EventIngested] = DocRangeField{Gte: "now-5m"}
 	} else {
-		docrange.Range[fn.EventIngested] = DocRangeField{Gte: ec.newestIngested.Format(time.RFC3339Nano)}
+		drf := DocRangeField{Gte: ec.newestIngested.Format(time.RFC3339Nano)}
+		if ec.search.FieldNames != nil && len(ec.search.FieldNames.TimestampFormat) > 0 {
```

Comment:
This is of course a matter of style and nothing is wrong with it -- but out of curiosity -- why `len(ec.search.FieldNames.TimestampFormat) > 0` instead of `ec.search.FieldNames.TimestampFormat == ""` which I think is more straightforward? :)